manifest.yaml

```diff
@@ -2,12 +2,15 @@ dist:
     module: github.com/observatorium/observatorium-otelcol
     name: observatorium-otelcol
     description: Observatorium OpenTelemetry Collector Distribution
-    version: 0.1.0
+    version: 0.1.1
     output_path: ./_build
-    otelcol_version: 0.10.0
+    otelcol_version: 0.13.0
 
 processors:
-  - gomod: github.com/observatorium/observatorium-otelcol/processors/authenticationprocessor v0.0.0
-    path: ./processors/authenticationprocessor
-  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/routingprocessor v0.10.0
-  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourcedetectionprocessor v0.10.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/k8sprocessor v0.13.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourcedetectionprocessor v0.13.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/routingprocessor v0.13.0
+
+replaces:
```

Comment:
Could you add a comment against all/each line here to explain why they are necessary, referencing any issues - so they can be cleaned up with the relevant issue is fixed?

Reply:
Done! Is the comment sufficient?

Reply:
Is there a github issue related to these? or are they going to stay this way?
Otherwise they are good.

Reply:
There's a general one: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/1217

Reply:
ok thanks